### PR TITLE
all Post methods

### PR DESCRIPTION
**Pull request by imsrish**

### CODE REVIEW

File: handlers/categoryHandler.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+router.post('/', (req, res) => {
+  let name = req.body.name;
+  let sponser = req.body.sponser;
+  let audience_size = req.body.audience_size;
+  let duration = req.body.duration;
+
+  let message = "";
+
+  if (name != "" && sponser != null && audience_size != "" && duration != "") {
```
Comment (JigyasuPrakash) [inline handlers/categoryHandler.js:33]: why sponser is compared with null and others with empty string?

File: handlers/organizerHandler.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  let poc_email = req.body.poc_email;
+  let comittee = req.body.comittee;
+
+  let message = "";
+
+  if (
+    poc_name != "" &&
+    comittee != "" &&
+    poc_email != "" &&
+    poc_contact != null
```
Comment (JigyasuPrakash) [inline handlers/organizerHandler.js:40]: why poc_contact is compared with null ?

File: handlers/venueHandler.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+router.post('/',(req,res)=> {
+    let name = req.body.name;
+    let approval_required= req.body.approval_required;
+    let capacity = req.body.capacity;
+    let location= req.body.location;
+
+    let message="";
+
+    if((name != "") && (approval_required != null) && (capacity != null) && (location != "")){
```
Comment (JigyasuPrakash) [inline handlers/venueHandler.js:29]: no consistency in field check constraints.. please rewrite this logic properly.

File: handlers/categoryHandler.js (hunk 1 above)
Comment (imsrish) [reply handlers/categoryHandler.js:33]: coz i thought true false is not supposed to be string

File: handlers/organizerHandler.js (hunk 2 above)
Comment (imsrish) [reply handlers/organizerHandler.js:40]: coz number hai, "" this will treat it as a string

File: handlers/venueHandler.js (hunk 3 above)
Comment (imsrish) [reply handlers/venueHandler.js:29]: "" for strings 
null for int and bolean

Comment (JigyasuPrakash) [reply handlers/venueHandler.js:29]: but your variable will always collect string data type from request.. have you typecasted to any specific datatype?

File: handlers/organizerHandler.js (hunk 2 above)
Comment (JigyasuPrakash) [reply handlers/organizerHandler.js:40]: but your variable will always collect string data type from request.. have you typecasted to any specific datatype?

File: handlers/categoryHandler.js (hunk 1 above)
Comment (JigyasuPrakash) [reply handlers/categoryHandler.js:33]: but your variable will always collect string data type from request.. have you typecasted to any specific datatype?

**Pull request merged**